consensus/polybft/polybft_config.go

```diff
@@ -16,7 +13,7 @@ import (
 
 const (
 	ConsensusName              = "polybft"
-	minNativeTokenParamsNumber = 4
+	minNativeTokenParamsNumber = 3
```

Comment:
is there an actual reason fir this change?

Reply:
Actually no, because we removed the whole native token functionality. Thanks for spotting this one, too.